decidim-core/app/packs/stylesheets/decidim/modules/_autocomplete.scss

```diff
@@ -0,0 +1,87 @@
+$custom-gray: #ededed;
```

Comment:
AutoComplete.js also seems to ship their own base CSS:
https://github.com/TarekRaafat/autoComplete.js/tree/master/dist/css

Did you try if we could use this as the base CSS and then make this custom CSS leaner, maybe just modifying the colors?

We want the component to feel fully integrated with the Decidim UI, so I don't mean we should change the look and feel, it's good as it is right now. But I'm just thinking can we remove some of this CSS.

They seem to provide few different stylesheets, so I'm not sure what's the purpose of each stylesheet. But just to check if it is possible to add less custom CSS, I think we should try them out.

The default CSS also seems to refer to some of their own SVG images, which we probably don't want, but maybe we can try to alternative CSS styles, if we are able to reduce the amount of CSS?